lms/djangoapps/discussion_api/serializers.py

```diff
@@ -25,9 +26,10 @@
 )
 from lms.lib.comment_client.comment import Comment
 from lms.lib.comment_client.thread import Thread
-from lms.lib.comment_client.user import User as CommentClientUser
+from lms.lib.comment_client.user import User as CommentClientUser, User
 from lms.lib.comment_client.utils import CommentClientRequestError
 from openedx.core.djangoapps.course_groups.cohorts import get_cohort_names
+from student.models import User
```

Comment:
We also don't need to import this we can use `DjangoUser` which is already available in this file.